### PR TITLE
Supported browsers

### LINKED ISSUE #79: Add information about supported browsers

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
closes #79

### CODE REVIEW

**Comment by MargaritaLoseva** [comment]
@kirovboris I've added only browsers we are testing. Should we add info about other browsers? Let's discuss this on the meeting.

File: documentation/guides/run-tests.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This topic describes how to execute tests and consists of the following sections
 
 ## Run Configurations
 
-TestCafe Studio can run tests in all desktop and mobile browsers (with or without a UI). It determines which browsers are installed on the local computer and automatically creates *run configurations* for them. You can view available configurations and switch between them in the **Run Configuration** drop-down menu.
+TestCafe Studio can run tests in desktop and mobile browsers, with or without a UI (see [Supported Browsers](../../references/supported-browsers.md)). TestCafe Studio determines which browsers are installed on the local computer and automatically creates *run configurations* for them. You can view available configurations and switch between them in the **Run Configuration** drop-down menu.
```
Comment (VasilyStrelyaev) [inline documentation/guides/run-tests.md]: > with or without a UI

maybe remove this remark because we do not support headless testing yet?

Comment (MargaritaLoseva) [reply documentation/guides/run-tests.md]: removed

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse Please review the changes

File: documentation/guides/record-tests/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The following sections provide detailed information about recording tests:
 
 ## Record Configurations
 
-You can record a test in any local browser. TestCafe detects browsers on the local computer and adds them to the **Record configuration** drop-down menu. Before recording a test, select the record configuration from the list.
+You can record tests in local browsers (see [Supported Browsers](../../references/supported-browsers.md)). TestCafe detects browsers on the local computer and adds them to the **Record configuration** drop-down menu. Before recording a test, select the record configuration from the list.
```
Comment (Nuarat) [inline documentation/guides/record-tests/README.md]: Why don't we include the link into original text? It'll be shorter & easier to read. And in this context it's logical to lead people to the browsers we're talking about.

> You can record tests in [local browsers](../../references/supported-browsers.md).

File: documentation/guides/run-tests.md [hunk 3]
```diff
@@ -14,11 +14,11 @@ This topic describes how to execute tests and consists of the following sections
 
 ## Run Configurations
 
-TestCafe Studio can run tests in all desktop and mobile browsers (with or without a UI). It determines which browsers are installed on the local computer and automatically creates *run configurations* for them. You can view available configurations and switch between them in the **Run Configuration** drop-down menu.
+TestCafe Studio can run tests in desktop and mobile browsers (see [Supported Browsers](../../references/supported-browsers.md)). TestCafe Studio determines which browsers are installed on the local computer and automatically creates *run configurations* for them. You can view available configurations and switch between them in the **Run Configuration** drop-down menu.
```
Comment (Nuarat) [inline documentation/guides/run-tests.md]: Same here, we can incorporate the link into the text:

> TestCafe Studio can run tests in [desktop and mobile browsers](../../references/supported-browsers.md).

File: documentation/references/supported-browsers.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Supported Browsers
+permalink: /documentation/references/supported-browsers.html
+---
+# Supported Browsers
+
+TestCafe Studio supports the following browsers:
```
Comment (Nuarat) [inline documentation/references/supported-browsers.md]: Why exactly don't we say "latest versions of..."?

File: documentation/references/supported-browsers.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+* Google Chrome: Stable, Beta, Dev and Canary
+* Internet Explorer (11+)
+* Microsoft Edge
+* Mozilla Firefox
+* Safari
+* Android browser
+* Safari mobile
```
Comment (Nuarat) [inline documentation/references/supported-browsers.md]: In the changes above we specify in which browsers we support running only, and in which we support recording.
I think this should be mentioned here as well.

What do you think?

Comment (helen-dikareva) [reply documentation/references/supported-browsers.md]: Yep, good not, I think we should say that we support recording only for desktop browsers.

File: documentation/guides/record-tests/README.md (hunk 2 above)
Comment (MargaritaLoseva) [reply documentation/guides/record-tests/README.md]: Usually, the link's text should correspond to the the topic's name. In your example, when a user clicks "local browsers", he/she expects to see a topic with the name "Local Browsers", but we have the "Supported Browsers" topic. 

Perhaps, it is an outdated rule.

Comment (MargaritaLoseva) [reply documentation/guides/record-tests/README.md]: Ok, I've discussed this with Vasily, and we decide to use Natasha's variant.

**Comment by MargaritaLoseva** [comment]
@Nuarat fpr

**Pull request merged**